client/pages/login/login.js

```diff
@@ -0,0 +1,52 @@
+// pages/login/login.js
+const qcloud = require('wafer2-client-sdk');
+const config = require('../../config');
+const app = getApp();
+
+function successfulLoginCallback(res) {
+  app.userInfo = res;
+  wx.switchTab({
+    url: '../index/index',
+  })
+  // qcloud.request({
```

Comment:
@ZhangYihanNus  I moved the code to this file and commented it. Please fix it.

Reply:
Noted！